### ISSUE TITLE
BP - ComboBox - wrong selected item

### ISSUE THREAD

**Issue by mirecad** [riganti/dotvvm#1604]
I have found out wrong unexpected behavior of business pack combox. 

When there are multiple items with same text in textbinding control does not return the correct selected item. The same behavior works correctly in classic combobox.

Demo: https://github.com/mirecad/DotvvmBpComboBoxBugDemo

1.  Create combobox items with the same label in viewmodel:
```
    public ComboBoxItemDto SelectedItem { get; set; }

    public List<ComboBoxItemDto> ComboBoxItems => new()
    {
        new ComboBoxItemDto()
        {
            Id = 1,
            Label = "Item 1",
            Detail = "Detail 1"
        },
        new ComboBoxItemDto()
        {
            Id = 2,
            Label = "Item 1",
            Detail = "Detail 2"
        }
    };
```

2. View:
```
        <bp:ComboBox DataSource="{value: ComboBoxItems}"
                     ItemKeyBinding="{value: Id}"
                     ItemTextBinding="{value: Label}"
                     SelectedValue="{value: SelectedItem}">
            <ItemTemplate>
                <div>
                    <div><b>{{value: Label}}</b></div>
                    <div>{{value: Detail}}</div>
                </div>
            </ItemTemplate>
        </bp:ComboBox>
           
        Selected item ID: {{value: SelectedItem.Id}}
```

If I select item ID 2, selected id is 2, but as soon as remove focus from combox the selected ID is switched to 1. It looks like KeyBinding is ignored.

![combobox](https://user-images.githubusercontent.com/13980038/222449120-213e06e9-66b4-428f-99fb-fc8ea8153141.gif)

**Comment by tomasherceg** [comment]
I believe we've fixed this exact issue in a recent release.
What is your version of the DotVVM Business Pack package?

**Comment by tomasherceg** [comment]
The issue should be fixed in DotVVM Business Pack `4.1.2` which we've just released. 

Closing the issue now. Feel free to reopen it if you find other problems with the control.
